examples/elect_leaders.c

```diff
@@ -0,0 +1,320 @@
+                               rd_kafka_error_string(err));
+                } else {
+                        printf("%% ElectLeaders succeeded for %s [%" PRId32
+                               "]\n",
+                               partition->topic, partition->partition);
+                }
+        }
+
+        return 0;
+}
+
+/**
+ * @brief Parse an integer or fail.
+ */
+int64_t parse_int(const char *what, const char *str) {
+        char *end;
+        unsigned long n = strtoull(str, &end, 0);
+
+        if (end != str + strlen(str)) {
+                fprintf(stderr, "%% Invalid input for %s: %s: not an integer\n",
+                        what, str);
+                exit(1);
+        }
+
+        return (int64_t)n;
+}
+
+static void cmd_elect_leaders(rd_kafka_conf_t *conf, int argc, char **argv) {
+        rd_kafka_t *rk;
+        char errstr[512];
+        rd_kafka_AdminOptions_t *options;
+        rd_kafka_event_t *event                     = NULL;
+        rd_kafka_topic_partition_list_t *partitions = NULL;
+        rd_kafka_ElectionType_t election_type;
+        rd_kafka_ElectLeaders_t *elect_leaders;
+        int i;
+        int retval = 0;
+
+        if ((argc - 1) % 2 != 0) {
+                usage("Invalid number of arguments");
+        }
+
+        election_type = parse_int("election_type", argv[0]);
+
+        argc--;
+        argv++;
+        if (argc > 0) {
+                partitions = rd_kafka_topic_partition_list_new(argc / 2);
+                for (i = 0; i < argc; i += 2) {
+                        rd_kafka_topic_partition_list_add(
+                            partitions, argv[i],
+                            parse_int("partition", argv[i + 1]));
+                }
+        }
+
+        elect_leaders = rd_kafka_ElectLeaders_new(election_type, partitions);
+
+        if (partitions) {
+                rd_kafka_topic_partition_list_destroy(partitions);
+        }
+
+        /*
+         * Create consumer instance
+         * NOTE: rd_kafka_new() takes ownership of the conf object
+         *       and the application must not reference it again after
+         *       this call.
+         */
+        rk = rd_kafka_new(RD_KAFKA_CONSUMER, conf, errstr, sizeof(errstr));
+        if (!rk)
+                fatal("Failed to create new consumer: %s", errstr);
+
+        /*
+         * Elect Leaders
+         */
+        queue = rd_kafka_queue_new(rk);
+
+        /* Signal handler for clean shutdown */
+        signal(SIGINT, stop);
+
+
+        options = rd_kafka_AdminOptions_new(rk, RD_KAFKA_ADMIN_OP_ELECTLEADERS);
+
+        if (rd_kafka_AdminOptions_set_request_timeout(
+                options, 10 * 1000 /* 10s */, errstr, sizeof(errstr))) {
+                fprintf(stderr, "%% Failed to set timeout: %s\n", errstr);
+                goto exit;
+        }
+
+        if (rd_kafka_AdminOptions_set_operation_timeout(
+                options, 10 * 1000 /* 10s */, errstr, sizeof(errstr))) {
```

Comment:
Same: Increase timeouts to 30s
```suggestion
                options, 30 * 1000 /* 10s */, errstr, sizeof(errstr))) {
```